banish/models.py

```diff
@@ -66,14 +69,52 @@ def is_current(self):
 
     class Meta:
         permissions = (("can_ban_user", "Can Ban User"),)
-        verbose_name = "Banishments"
+        verbose_name = "Banishment"
         verbose_name_plural = "Banishments"
         db_table = 'banishments'
 
 
+class Whitelist(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
I just tried a syncdb after removing both, and indeed, both tables still exist and are populated with my test bans and whitelists.